tests/suites/0_stateless/02_00007_function_aggreator_on_empty.result

```diff
@@ -0,0 +1,16 @@
+
```

Comment:
```
/// Get the value at the given row in an array as a String.
///
/// Note this function is quite inefficient and is unlikely to be
/// suitable for converting large arrays or record batches.
pub fn array_value_to_string(column: &array::ArrayRef, row: usize) -> Result<String> {
    if column.is_null(row) {
        return Ok("".to_string());
    }
```

Seems the behavior in arrow `array_value_to_string` is wrong.

Reply:
Ok, In some databases, empty string is equal to null , like oracle.